pkg/runner/job_runner.go

```diff
@@ -104,7 +104,7 @@ func (jr *JobRunner) Run(j *job.Job) ([][]*job.Report, []*job.Report, error) {
 				// the Acquire semantic is synchronous, so that the implementation
 				// is simpler on the user's side. We run it in a goroutine in
 				// order to use a timeout for target acquisition.
-				targets, err := bundle.TargetManager.Acquire(j.StateCtx, j.ID, bundle.AcquireParameters, tl)
+				targets, err := bundle.TargetManager.Acquire(j.StateCtx, j.ID, j.TargetManagerAcquireTimeout, bundle.AcquireParameters, tl)
```

Comment:
I think this should be `j.TargetManagerAcquireTimeout + config. LockRefreshTimeout`, right? Just `j.TargetManagerAcquireTimeout` will race with the `Lock` below, so we need a bit more

Reply:
Sounds fair.